src/overlays/actors/ovl_Obj_Grass_Carry/z_obj_grass_carry.c

```diff
@@ -61,89 +61,91 @@ static ColliderCylinderInit sCylinderInit = {
     { 10, 44, 0, { 0, 0, 0 } },
 };
 
-s16 D_809ABBFC = 0;
-s16 D_809ABC00 = 0;
-s16 D_809ABC04 = 0;
-s16 D_809ABC08 = 0;
+static s16 sRotSpeedXTarget = 0;
+static s16 sRotSpeedX = 0;
+static s16 sRotSpeedYTarget = 0;
+static s16 sRotSpeedY = 0;
 
-Vec3f D_809ABC0C[] = {
+static Vec3f sUnitDirections[] = {
     { 0.0f, 0.7071f, 0.7071f },
     { 0.7071f, 0.7071f, 0.0f },
     { 0.0f, 0.7071f, -0.7071f },
     { -0.7071f, 0.7071f, 0.0f },
 };
 
-s16 D_809ABC3C[] = { 0x6C, 0x66, 0x60, 0x54, 0x42, 0x37, 0x2A, 0x26 };
+static s16 sFragmentScales[] = { 108, 102, 96, 84, 66, 55, 42, 38 };
 
 static InitChainEntry sInitChain[] = {
     ICHAIN_F32_DIV1000(gravity, -3200, ICHAIN_CONTINUE),
     ICHAIN_F32_DIV1000(terminalVelocity, -17000, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 400, ICHAIN_STOP),
 };
 
-void func_809AAE60(ObjGrassCarry* this) {
+void ObjGrassCarry_UpdateVelY(ObjGrassCarry* this) {
     this->actor.velocity.y += this->actor.gravity;
     if (this->actor.velocity.y < this->actor.terminalVelocity) {
         this->actor.velocity.y = this->actor.terminalVelocity;
     }
 }
 
-void func_809AAE94(Vec3f* arg0, f32 arg1) {
-    arg1 += ((Rand_ZeroOne() * 0.2f) - 0.1f) * arg1;
+void ObjGrassCarry_RandScaleVecToZero(Vec3f* vel, f32 scale) {
+    scale += ((Rand_ZeroOne() * 0.2f) - 0.1f) * scale;
 
-    arg0->x -= (arg0->x * arg1);
-    arg0->y -= (arg0->y * arg1);
-    arg0->z -= (arg0->z * arg1);
+    vel->x -= (vel->x * scale);
+    vel->y -= (vel->y * scale);
+    vel->z -= (vel->z * scale);
```

Comment:
```suggestion
    vel->y -= vel->y * scale;
    vel->z -= vel->z * scale;
```
Probably don't need these brackets